### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by shivam1192**
Updated readme.md by using American-English words. 
Also, update minor grammatical errors.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,15 +9,15 @@ The Select control for [React](https://reactjs.com). Initially built for use in
 
 See [react-select.com](https://www.react-select.com) for live demos and comprehensive docs.
 
-React Select is funded by [Thinkmill](https://www.thinkmill.com.au) and [Atlassian](https://atlaskit.atlassian.com). It represents a whole new approach to developing powerful React.js components that _just work_ out of the box, while being extremely customisable.
+React Select is funded by [Thinkmill](https://www.thinkmill.com.au) and [Atlassian](https://atlaskit.atlassian.com). It represents a whole new approach to developing powerful React.js components that _just work_ out of the box while extremely customizable.
```
Comment (geekabhinav) [inline README.md:12]: `while being extremely customisable` is the correct English. Your edit makes it sound weird.

**Review by geekabhinav — CHANGES_REQUESTED** [review]
Stop raising spam PRs for HacktoberFest 😡

**Comment by shivam1192** [comment]
Huh? I had already completed with hacktoberfest challenge. I raise this pr so as to familiarise with contribution guide and bots. Between american-english is more preferable in tech industry as British-english, I suppose.
Grammerly premium mention this changes.

**Comment by cseas** [comment]
Spam PR

**Pull request closed without merging**